src/EditorFeatures/CSharp/Outlining/CSharpOutliningService.cs

```diff
@@ -40,6 +40,7 @@ internal class CSharpOutliningService : AbstractOutliningService
             builder.Add<MethodDeclarationSyntax, MethodDeclarationOutliner, MetadataAsSource.MethodDeclarationOutliner>();
             builder.Add<NamespaceDeclarationSyntax, NamespaceDeclarationOutliner>();
             builder.Add<OperatorDeclarationSyntax, OperatorDeclarationOutliner, MetadataAsSource.OperatorDeclarationOutliner>();
+            builder.Add<ParenthesizedLambdaExpressionSyntax, ParenthesizedLambdaExpressionOutliner>();
```

Comment:
Wait, so we have the code for this, we just forgot to add it to the builder when we moved away from MEF? :sob: